src/main/java/org/apache/brooklyn/cloudfoundry/entity/VanillaCloudFoundryApplication.java

```diff
@@ -72,6 +69,11 @@
             "cloudFoundry.application.host", "Host or sub-domain for the application, if " +
                     "this value is empty the application name will be used like the host");
 
+    @SetFromFlag("services")
+    ConfigKey<List<Object>> SERVICES = ConfigKeys.newConfigKey(new TypeToken<List<Object>>() {
```

Comment:
This is the last big question about service management (I think). I tried to explain the status here https://github.com/kiuby88/brooklyn-cloudfoundry/pull/19#issuecomment-240345458